packages/components/src/AppBar/AppBar.tsx

```diff
@@ -13,7 +13,7 @@ import styled from 'styled-components'
 import { grid } from '../grid'
 import { Stack } from '../Stack'
 import { Text } from '../Text'
-import { useWindowSize } from './useWindowSize'
+import { useWindowSize } from '@opencrvs/client/src/hooks/useWindowSize'
```

Comment:
client uses components as a dependency so we can't have components importing something from client.

Reply:
The tests are also failing because of this

Reply:
Should we also add the hooks directory to the components package and then move the useWindowSize hook there and use it in the client package?